airflow/bin/cli.py

```diff
@@ -1043,7 +1043,7 @@ def serve_logs(args):
     flask_app = flask.Flask(__name__)
 
     @flask_app.route('/log/<path:filename>')
-    def serve_logs(filename):
+    def serve_logs(filename):  # pylint: disable=unused-variable
```

Comment:
Why do you need to disable this? `filename` is used in line 1050.

Reply:
```
airflow/bin/cli.py:1046:4: W0612: Unused variable 'serve_logs' (unused-variable)
```

Reply:
Yeah. Pylint sometimes gets crazy.

Reply:
Have we reported this as a bug to pylint?

Perhaps we should disable that pylint error globally since Flake8 has a (better?) check for this already

Reply:
Except I guess it doesn't have a better check as some of the other fixes here weren't caught by flake8.

Reply:
The problem is only in the content of the message. Pylint detected unused functions and marked them as variables. Flake8 does not detect unused functions. 

I did not report this error to pylint.

Reply:
Ah it's on the function, not on the `filename` variable itself (which I guess would have been unused-argument)

https://github.com/PyCQA/pylint/issues/2842 seems to cover it, and I agree with that bug that it can't be handled. My only confusion was I would have expected the error to be unused-function.